pkgs/development/libraries/mapbox-gl-native/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, mkDerivation, fetchFromGitHub, cmake, pkg-config
+, qtbase, curl, libuv, glfw3 }:
+
+mkDerivation rec {
+  pname = "mapbox-gl-native";
+  version = "2020.06.07";
+
+  src = fetchFromGitHub {
+    owner = "rinigus";
```

Comment:
I don't feel great about putting a fork of an active project info global namespace.

Reply:
I've replaced it with the upstream version (Turns out there aren't any relevant differences)